lib/subscription-connection.js

```diff
@@ -79,7 +81,8 @@ module.exports = class SubscriptionConnection {
 
     const sc = new SubscriptionContext({
       fastify: this.fastify,
-      pubsub: this.subscriber
+      pubsub: this.subscriber,
+      context: this.context
```

Comment:
I think the context shouldn't be added here, but to the actual GraphQL context object (line 95). If we add it here the context object can be accessed in the subscriber function via the `pubsub` object (`context.pubsub.context`). If it's spread on the GraphQL context object it seems better I think (also in line with your original example in #151 )

Reply:
I agree! Implemented this in https://github.com/mcollina/fastify-gql/pull/153/commits/c7d4cc846181ed5711b412267926ec4bcb162091